### ISSUE TITLE
An actual Floating layout?

### ISSUE THREAD

**Issue by tamis-laan** [qtile/qtile#1444]
Hey is it possible to have an actual floating layout, as I understand is Floating is not really a layout but configures the floating layout state. 

However I would like to be able to switch to an actual Floating layout where all windows are floating and the position and size of the windows are remembered when switching layouts back and forth. 

Is this possible?

**Comment by ramnes** [comment]
No, it's not possible as of today. You can already add an instance of `libqtile.layout.Floating` in your configuration `layouts` variable, but as you noticed, any modification of a window's size or position will result in marking that window as floating, even when you switch to another layout.

What you describe would be a really nice feature though, and I would love to see a pull request on this.
